PYTHON_VERSION

```diff
@@ -0,0 +1 @@
+3.10.10
```

Comment:
was this meant to be added?  Didn't mention in the readme thats all.

Reply:
it was, allows https://github.com/timeoutdigital/timeout-tools to know what python to setup for a cloned repo